helm_deploy/django-app/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
+replicaCount: 3
```

Comment:
I'd leave the default here as `1` - then we can use this as an example of a user-definable chart value, where the app can be scaled up with a `helm upgrade`.

Reply:
This allows us to perform zero outage deploys via circle. I suppose we can just overwrite this in the circle config.

Reply:
done

Reply:
yeah - it's just a question of what the default value is here. Helm charts most commonly [have a default of `1`](https://github.com/kubernetes/charts/blob/master/stable/sentry/values.yaml#L11) as not every environment will want a high availability configuration. In our case we do want > 1 replica, but that just means we need to pass `--set replicaCount=3` in `helm install` in circle, or have it set in a `values.yml` file for circle.